toolchain/sem_ir/cpp_global_var.h

```diff
@@ -0,0 +1,78 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_SEM_IR_CPP_GLOBAL_VAR_H_
+#define CARBON_TOOLCHAIN_SEM_IR_CPP_GLOBAL_VAR_H_
+
+#include "common/hashing.h"
+#include "common/ostream.h"
+#include "toolchain/sem_ir/clang_decl.h"
+#include "toolchain/sem_ir/ids.h"
+
+namespace Carbon::SemIR {
+
+// A key describing a C++ global variable imported into Carbon, identified by
+// its name.
+struct CppGlobalVarKey : public Printable<CppGlobalVarKey> {
+  auto Print(llvm::raw_ostream& out) const -> void {
+    out << "{name_id: " << name_id << "}";
+  }
+
+  auto operator==(const CppGlobalVarKey& rhs) const -> bool {
+    return name_id == rhs.name_id;
+  }
+
+  // Hashing for CppGlobalVarKey. See common/hashing.h.
+  friend auto CarbonHashValue(const CppGlobalVarKey& value, uint64_t seed)
+      -> HashCode {
+    // Manual hashing support is required because this type has tail padding in
+    // 64-bit compilations.
+    return HashValue(value.name_id, seed);
+  }
+
+  // The name of the variable.
+  EntityNameId name_id;
```

Comment:
```suggestion
  EntityNameId entity_name_id;
```
`NameId` is a different thing

Reply:
Done.